src/ts/canvas.ts

```diff
@@ -13,24 +13,6 @@ export function clearCanvas() {
   canvasContext.fillRect(0, 0, canvas.width, canvas.height);
 }
 
-export function fillRect(
```

Comment:
Can you keep this function so we can use it in the future if needed? It's helpful for extending the function to support rotation

Reply:
Bump